### PR TITLE
Debriefer error checks, add long trip check #2151

### PR DESCRIPTION
**Pull request by Mak-NOAA**
Changes related to https://github.com/nwfsc-fram/boatnet/issues/2151
Adding long trip check

### CODE REVIEW

File: util/tripChecks.ts [hunk 1]
```diff
@@ -179,4 +180,26 @@ function runObsLogbookMissingCheck(tripErrorDoc: WcgopTripError, trip: any) {
...
+        status: StatusType.valid,
+        errorItem: 'Trip Length',
+        errorValue: moment(trip.departureDate).diff(trip.returnDate).toString(),
+        notes: '',
+        legacy:{
+            checkCode : 500 
+        }
+    };
+
+    if ( trip.fishery!=21 && moment(trip.departureDate).diff(trip.returnDate)>10 )
```
Comment (sethgerou-noaa) [inline util/tripChecks.ts:201]: The current WCGOP trip model has fishery?: Fishery - meaning it's storing the entire fishery doc, so we could use trip.fishery.lookupValue (to compare with 21), but what appears to be in current Couch WCGOP trip docs is 
  "fishery": {
    "description": "OR Pink Shrimp",
    "_id": "f129bb19a290d658401c49cf3e9ae312"
  },
as Couch can't do joins and we'd have to do another query to get any info about that fishery._id, I'd suggest we discuss with Nick - seems like storing the entire fishery would be better.

Comment (sethgerou-noaa) [reply util/tripChecks.ts:201]: actually... looks like the Fishery.lookupValue values are the same as the description - which I'd assume means the numbers are no longer used - I see 21 as a legacy value - but if it's in legacy, we shouldn't be using it.

Maybe it's sufficient to switch to trip.fishery.description !== "Mothership Catcher-Vessel" and leave the trip docs as is.

Comment (Mak-NOAA) [reply util/tripChecks.ts]: @sethgerou-noaa  see the commit below....I switched it to trip.fishery.description !== "Mothership Catcher-Vessel"  but do you think it's a bad idea to use fisher.legacy.lookupVal!=21?

Comment (sethgerou-noaa) [reply util/tripChecks.ts]: Yeah, I do think it's a bad idea to use anything in legacy.  If we want to use a code, it shouldn't be in legacy.
If you feel strongly about the value of lookup code vs description, I'd suggest we discuss adding in back into the fishery model during the Design discussion on Wednesday.

**Review by sethgerou-noaa — APPROVED** [review]

**Pull request merged**